app/preview/TemplateParams.scala

```diff
@@ -2172,14 +2178,18 @@ object TemplateParams2 {
       "eori"                      -> "GB123456789000000"
     ),
     "mods_import_declaration_cy" -> Map(
-      "emailTo"                   -> "BorderForce",
+      "emailTo"                   -> "joebloggs@email.com",
       "recipientName_FullName"    -> "Joe Bloggs",
-      "declarationReference"      -> "ABC123",
+      "declarationReference"      -> "XADST0000010002",
       "dateOfDeclaration"         -> "11 November 2020, 4:22pm",
       "goodsCategory_0"           -> "shoes",
       "goodsQuantity_0"           -> "10",
       "goodsCountry_0"            -> "Sbaen",
       "goodsPrice_0"              -> "150, Ewro (EUR)",
+      "goodsCategory_1"           -> "hats",
+      "goodsQuantity_1"           -> "100",
+      "goodsCountry_1"            -> "Mecsico",
```

Comment:
same as above , this should be changed to `goodsProducedInEu_1` if its import

Reply:
Guess the examples will look better with new question so ok.